src/main/java/utils/ConfigReader.java

```diff
@@ -0,0 +1,19 @@
+package utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigReader {
```

Comment:
I think we will not use this wrapper because we will have more than one configuration files to be continuously used, which will cause that every time we want to read a property from a configuration file to go and create a new properties object that will initialize a new <FileInputStream> and open it.  I am recommending the use of a private properties instance in the place you are going to use the configuration file to always have a <FileInputStream> always opened. I think from my try in search-engine you will need to access each of the configuration files in only one place which will make this design approach good.

Reply:
I think each config file is used in only one place, so we will never need to continuously initialize a new properties object.
I agree with you that we can have a singleton instance for every config properties, I can do this when I complete the commands implementation.

Reply:
Ok sorry, I thought that you will make `ConfigReader` singleton.